PWGHF/HFC/TableProducer/correlatorDMesonPairsTesting.cxx

```diff
@@ -311,11 +313,16 @@ struct HfCorrelatorDMesonPairsTesting {
   /// Fill counters for D0 and D0bar
   /// \param selectedD0Candidates contains all D0 candidates
   template <typename T>
-  void GetCountersPerEvent(const T& selectedD0Candidates)
+  void GetCountersPerEvent(const T& selectedD0Candidates, const bool& selectSignalRegionOnly)
```

Comment:
@atavirag For my information, why is this addition needed?

Reply:
Indeed there is no need to have `selectSignalRegionOnly` as argument of the function, this has slipped my eyes when looking at the code. @atavirag can you remove it in another PR? Thanks!

Reply:
Hi, sorry, I just made the PR with the correction. It's #4732

Reply:
Thanks